app/src/main/java/com/example/mycalculator/MainActivity.java

```diff
@@ -93,8 +92,40 @@ public void onClick(View v) {
         binding.buttonFunctionErase.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                presenter.numberPressed(StringUtils.EMPTY_STRING);
+                presenter.erase();
             }
         });
+        binding.buttonFunctionSum.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                presenter.operatorPressed(binding.buttonFunctionSum.getText().toString());
+
+            }
+        });
+        binding.buttonFunctionMinus.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                presenter.operatorPressed(binding.buttonFunctionMinus.getText().toString());
+            }
+        });
+        binding.buttonFunctionMultiplication.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                presenter.operatorPressed(binding.buttonFunctionMultiplication.getText().toString());
+            }
+        });
+        binding.buttonFunctionSplit.setOnClickListener(new View.OnClickListener() {
```

Comment:
you can rename this as divide instead of split. Split is used for a different meaning

Reply:
Done. Thx